sdcm/sct_config.py

```diff
@@ -819,6 +819,21 @@ class SCTConfiguration(dict):
 
         dict(name="logs_transport", env="SCT_LOGS_TRANSPORT", type=str,
              help="How to transport logs: rsyslog or ssh"),
+
+        dict(name="collect_logs", env="SCT_COLLECT_LOGS", type=boolean,
+             help="Collect logs from instances and sct runner"),
+
+        dict(name="execute_post_behavior", env="SCT_EXECUTE_POST_BEHAVIOR", type=boolean,
+             help="Run post behavior actions in sct teardown step"),
+
+        dict(name="post_behavior_db_nodes", env="SCT_POST_BEHAVIOR_DB_NODES", type=str,
```

Comment:
if we are introducing a those new options, what about the old `failure_post_behavior` ? it would be removed ? or automatically mapped to those ?

Reply:
i think old one should be totally replaced with new ones
Current design, if new parameters are not in config file, then old config is used.